lib/modules/datasource/python-version/readme.md

```diff
@@ -0,0 +1 @@
+This datasource returns Python releases from the [python.org API](https://www.python.org/api/v2/downloads/release/).
```

Comment:
Please also add info that it reads additional metadata from endoflife and github api